BrainPortal/app/controllers/tasks_controller.rb

```diff
@@ -1681,4 +1683,93 @@ def self.compact_hash(hash)
     end
   end
 
+
+  public
+
+  # Tasks-specific batch filter; filters by batch id to return a list
+  # of subtasks.
+  # Note that this filter uses Scope::Filter's *value* attribute to hold the
+  # id to check against, and that the *attribute* attribute is
+  # statically set to 'batch_id' (as this filter will only ever filter on a task's
+  # batch ).
+  class BatchFilter < Scope::Filter
+
+    # Create a new blank BatchFilter. Only present to pre-set *attribute*.
+    def initialize
+      @attribute = 'batch_id'
+    end
+
+    # Nice string representation of this filter for +pretty_scope_filter+.
+    def to_s
+      "Batch Id: ##{@value.to_s.humanize}"
+    end
+
+    # The methods below are TagFilter specific versions of the Scope::Filter
```

Comment:
Should be `BatchFilter` not TagFilter. Same error line l1604 where it should be `Status Filter`